.devcontainer/devcontainer.json

```diff
@@ -24,5 +24,8 @@
     "hostRequirements": {
         "memory": "8gb",
         "cpus": 4
-    }
+    },
+    "mounts": [
+        "source=/opt/docker/data/jellyfin/config10.9.11/metadata/,target=/workspaces/jellyfin/TestData/metadata,type=bind,consistency=cached"
+    ]
```

Comment:
I believe the whole commit that introduces this should be dropped altogether.

This it's only relevant for your own testing environment and it's polluting the commit history with unintended changes.

Reply:
As discussed in chat, branch will be squash merged.